br/pkg/backup/client.go

```diff
@@ -50,6 +50,8 @@ const (
 	MaxResolveLocksbackupOffSleepMs = 600000
 
 	IncompleteRangesUpdateInterval = time.Second * 15
+
+	RangesSentThreshold = 30000000
```

Comment:
should we make it configuration? will it block backup speed, if there is too many subrangs.